.chglog/config.yml

```diff
@@ -0,0 +1,28 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/aws/aws-dotnet-deploy
+options:
+  commits:
+    # filters:
+    #   Type:
+    #     - feat
+    #     - fix
+    #     - perf
+    #     - refactor
```

Comment:
Do you want them to keep as comments?

Reply:
Yes, I left them there to remind us that we can filter out certain types if we have that discussion before the next release